fluentui_core/src/main/java/com/microsoft/fluentui/compose/ModalPopup.kt

```diff
@@ -183,75 +302,48 @@ private class ModalWindow(
                 flags
             } else flags or WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS
         }
-
-    private var content: @Composable () -> Unit by mutableStateOf({})
-
-    override var shouldCreateCompositionOnAttachedToWindow: Boolean = false
-        private set
-
-    @Composable
-    override fun Content() {
-        content()
-    }
-
-    fun setCustomContent(
-        parent: CompositionContext? = null,
-        content: @Composable () -> Unit
-    ) {
-        parent?.let { setParentCompositionContext(it) }
-        this.content = content
-        shouldCreateCompositionOnAttachedToWindow = true
     }
+}
 
-    fun show() {
-        windowManager.addView(this, params)
-    }
+@VisibleForTesting
+internal interface PopupLayoutHelper {
+    fun getWindowVisibleDisplayFrame(composeView: View, outRect: Rect)
+    fun setGestureExclusionRects(composeView: View, width: Int, height: Int)
+    fun updateViewLayout(
+        windowManager: WindowManager,
+        popupView: View,
+        params: ViewGroup.LayoutParams
+    )
+}
 
-    fun dismiss() {
-        this.setViewTreeLifecycleOwner(null)
-        setViewTreeSavedStateRegistryOwner(null)
-        composeView.viewTreeObserver.removeOnGlobalLayoutListener(this)
-        windowManager.removeViewImmediate(this)
+private open class PopupLayoutHelperImpl : PopupLayoutHelper {
```

Comment:
formatting

Reply:
It is formatted, the popup layout helper methods are defined outside the class